sklearn_numba_dpex/common/kernels.py

```diff
@@ -100,6 +100,34 @@ def broadcast_division(dividend_array, divisor_vector):
     return broadcast_division[global_size, work_group_size]
 
 
+@lru_cache
+def make_broadcast_ops_1d_2d_axis1_kernel(size0, size1, ops, work_group_size):
+    """
+    ops must be a function that will be interpreted as a dpex.func and is subject to
+    the same rules. It is expected to take two scalar arguments and return one scalar
+    value. lambda functions are advised against since the cache will not work with lamda
+    functions."""
+
+    global_size = math.ceil(size1 / work_group_size) * work_group_size
+    ops = dpex.func(ops)
+
+    # NB: inplace. # Optimized for C-contiguous array and for
+    # size1 >> preferred_work_group_size_multiple
+    @dpex.kernel
+    def broadcast_ops(left_operand_array, right_operand_vector):
```

Comment:
I think it's worth indicating that the left operand is modified in place and that the right one isn't modified at all.